### PR TITLE
[23288] [Accessibility] Tabschritte in Tabelle können nicht übersprungen werden

### PR DESCRIPTION
**Pull request by HDinger**
This inserts a skip link to enable blind users to jump over the WP table with one click.

TODOs:
- [x] There is a link before the table. Clicking on the link will set the focus to the pagination area below.
- [x] The link is only visible on focus.
- [x] The screenreader reads the link correctly link.

https://community.openproject.com/work_packages/23288/activity

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hdinger-fix-23288-tabschritte-in-tabelle-koennen-nicht-ueberspr.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://hdinger-fix-23288-tabschritte-in-tabelle-koennen-nicht-ueberspr.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: frontend/app/components/routing/wp-list/wp-list.controller.ts [hunk 1]
```diff
@@ -159,6 +159,18 @@ function WorkPackagesListController($scope,
     AuthorisationService.initModelAuth("query", meta.query._links);
   }
 
+  $scope.setAnchorToNextPage = function () {
+    anchor = '';
+    if(jQuery('.pagination-number').last().hasClass('-current')) {
```
Comment (HDinger) [inline frontend/app/components/routing/wp-list/wp-list.controller.ts]: @oliverguenther is there a more elegant way to find out wether the last pagination element is reached?

Comment (oliverguenther) [inline frontend/app/components/routing/wp-list/wp-list.controller.ts]: Sorry to not have gotten back to you. I'll have a look whether we can add a more specific class to the pagination elements

**Comment by oliverguenther** [comment]
@HDinger I've added a separate tabbable span when neither previous nor next button is shown to make sure the user can always skip the table.

With that, we can also use focus() on either of these three selectors. It doesn't requite mangling the location hash.

Other than that, I've only changed the existing jump label to be more explicit and added a few aria-labels here and there. Could you take another look if all works as expected?

anim
anim2

**Comment by oliverguenther** [comment]
@HDinger  I've tested the navigation with IE11/Jaws 17 using the hotfix on the other PR. Feel free to merge this from my side after travis.

**Comment by HDinger** [comment]
@oliverguenther works good. Only the case when there is no pagination doesn't seem to work correctly. If I get it correctly there should be the label 'You are on the only page'. But there is still 'jump to table pagination'.

**Comment by HDinger** [comment]
ok, forget it. I understood it wrong. The label is down at the pagination not on the top. 
I will have a look at the failing tests. Once travis is green we can merge from my side, too.

**Comment by oliverguenther** [comment]
I'm almost done fixing both tests, the karma test is only a whitespace issue due to the additional span

**Comment by HDinger** [comment]
👍

**Comment by oliverguenther** [comment]
Done. Travis should agree now!

**Comment by HDinger** [comment]
Great! Thanks a lot!

**Pull request merged**